### PR TITLE
Better menu for 'choose NFT'

### PR DESCRIPTION
**Pull request by osdnk**
Fixes RNBW-####
Figma link (if any):

## Screen recordings / screenshots

My ENS Names

## Final checklist

- [ ] Assigned individual reviewers?
- [ ] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [ ] Did you test both iOS and Android?
- [ ] If your changes are visual, did you check both the light and dark themes?
- [ ] Added e2e tests? If not, please specify why
- [ ] If you added new critical path files, did you update the CODEOWNERS file?
- [ ] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

File: src/screens/ENSIntroSheet.tsx [hunk 1]
```diff
@@ -292,9 +267,7 @@ export default function ENSIntroSheet() {
                           {nonPrimaryDomains?.length > 0 ? (
                             <ContextMenuButton
                               menuConfig={menuConfig}
-                              {...(android
-                                ? { onPress: onPressAndroidActions }
-                                : {})}
+                              {...(android ? { handlePressMenuItem } : {})}
```
Comment (skylarbarrera) [inline src/screens/ENSIntroSheet.tsx:270]: o.O

**Review by skylarbarrera — APPROVED** [review]
good

File: src/screens/ENSIntroSheet.tsx (hunk 1 above)
Comment (osdnk) [reply src/screens/ENSIntroSheet.tsx:270]: good

Comment (tchayen) [reply src/screens/ENSIntroSheet.tsx:270]: good

**Review by estebanmino — APPROVED** [review]
:cheff-kiss: https://www.loom.com/share/1b7b442270754cd2aa0d40b85509d437

**Pull request merged**